tests/test_spherePoint.py

```diff
@@ -544,6 +527,25 @@ def testBearingToValue(self):
                 else:
                     self.assertTrue(math.isnan(bearing.asRadians()))
 
+    def testBearingToValueSameLongitude(self):
+        """Test that bearingTo() returns +/- 90 for two points on the same longitude
+        """
+        for longDeg in (0, 55, 270):
+            for lat0Deg in (-90, -5, 0, 44, 90):
+                sp0 = SpherePoint(longDeg, lat0Deg, degrees)
+                for lat1Deg in (-90, -41, 1, 41, 90):
+                    if lat0Deg == lat1Deg:
+                        continue
+                    sp1 = SpherePoint(longDeg, lat1Deg, degrees)
+                    if sp0.atPole() and sp1.atPole():
+                        # the points are at opposite poles; any bearing may be returned
+                        continue
+                    bearing = sp0.bearingTo(sp1)
+                    if lat1Deg > lat0Deg:
+                        self.assertAnglesAlmostEqual(bearing, 90 * degrees)
+                    else:
+                        self.assertAnglesAlmostEqual(bearing, -90 * degrees)
+
```

Comment:
Test bearing from a pole to a point not on the same longitude? As I understand it, the answer should be the difference in longitudes minus 90 degrees for the north pole, and the difference plus 90 degrees for the south pole.